discmoji/bot.py

```diff
@@ -90,9 +90,16 @@ def _all_guilds_setter(self):
             self.all_guilds: List[Guild] = []
             self.all_guilds.append(Guild(self._gateway_client.current_payload.data))
     
-    
-    
+    async def update_all_objects(self):
```

Comment:
this is awful. just update real-time. outline how your caching will work \:shrug: